src/kirby/components/chip/chip.component.scss

```
@@ -0,0 +1,17 @@
+@import "../../scss/web-imports";
+$min-chip-width: 50px;
+
+:host {
+  white-space:nowrap;
+  padding: 0 25px;
+  height: 30px;
+  font-size: font-size('s');
+  line-height: line-height('xl');
+  border-radius: 25px;
+  margin-right: 5px;
+  min-width: $min-chip-width;
+  cursor: pointer;
+  &:hover,&.isSelected {
```

Use css naming convention:
```suggestion
  &:hover,&.is-selected {
```

---

Done